src/test/run-tests.sh

```diff
@@ -3,10 +3,40 @@
 SCRIPTDIR="$(dirname ""$0"")"
 
 MOZIOT_HOME="${MOZIOT_HOME:=${HOME}/.mozilla-iot}"
-MOZIOT_HOME="${MOZIOT_HOME}/test"
-if [ ! -f "${MOZIOT_HOME}/ssl/certificate.pem" ]; then
-  export MOZIOT_HOME
-  ${SCRIPTDIR}/../../tools/make-self-signed-cert.sh
+MOZIOT_TEST_HOME="${MOZIOT_HOME}/test"
+if [ ! -f "${MOZIOT_TEST_HOME}/ssl/certificate.pem" ]; then
+  MOZIOT_HOME="${MOZIOT_TEST_HOME}" ${SCRIPTDIR}/../../tools/make-self-signed-cert.sh
 fi
 
+# Check to see if the adapt module is installed
+if ! python -c 'import adapt'; then
```

Comment:
No need to use sudo here. `pip install --user git+https://github.com/mycroftai/adapt#egg=adapt-parser`